modules/ROOT/pages/dataweave-system-properties.adoc

```diff
@@ -202,45 +169,44 @@ output application/json
-
-* Type: `Number`
-* Default: `256`
-
-
-//com.mulesoft.dw.xml.supportDTD//
-|`com.mulesoft.dw.xml.supportDTD`|
-Controls whether DTD handling is enabled or disabled. When this property is set to `false`, DataWeaves skips processing of both internal and external subsets. Note that the default for this property changed from `true` to `false` in Mule version 4.3.0-20210427, which includes the May, 2021 patch of DataWeave version 2.3.0.
-
 * Type: `Boolean`
 * Default: `false`
-
-
-//com.mulesoft.dw.track.cursor.close//
-|`com.mulesoft.dw.track.cursor.close`|
-When set to `true`, tracks the stack trace from which the `CursorProvider#close()` method is called.
-Use this property for troubleshooting, for example, if `CursorProvider#openCursor()` is called on a cursor that is already closed.
-
+Available for language levels: 2.3, 2.4
+//com.mulesoft.dw.workingdirectory.base//
+|`com.mulesoft.dw.workingdirectory.base`|
+Specifies the base directory in which temporary files will be written
+* Type: `String`
+* Default: `/var/folders/8f/5jqs1kh97k16dzsmxxph2m5c0000gp/T/`
+//com.mulesoft.dw.workingdirectory.delete_sync//
+|`com.mulesoft.dw.workingdirectory.delete_sync`|
+If set, file deletion is done synchronously
 * Type: `Boolean`
 * Default: `false`
-
-
-//com.mulesoft.dw.verbose//
-|`com.mulesoft.dw.verbose`|
-Displays filtered stack traces when this property is set to `true`. Does not display stack traces when set to `false`.
-
+//com.mulesoft.dw.workingdirectory.max_output_buffer_size//
+|`com.mulesoft.dw.workingdirectory.max_output_buffer_size`|
+Specifies the maximum size of a file to be written in bytes
+* Type: `Long`
+* Default: `-1`
+//com.mulesoft.dw.workingdirectory.tracking//
+|`com.mulesoft.dw.workingdirectory.tracking`|
+If set, then each script generates tmp files in their own directory
 * Type: `Boolean`
 * Default: `false`
-
-
-//com.mulesoft.dw.xml_reader.honourMixedContentStructure
+//com.mulesoft.dw.xml_reader.honourMixedContentStructure//
 |`com.mulesoft.dw.xml_reader.honourMixedContentStructure`|
-When this property is set to `true`, DataWeave retains a mixed-content structure instead of grouping text with mixed content into a single text field.
-
+When this property is set to `true`, DataWeaves honour mixed content structure instead of grouping text mixed content in a single text field.
 * Type: `Boolean`
 * Default: `false`
+Available for language levels: 2.4
+//com.mulesoft.dw.xml_reader.parseDtd//
+|`com.mulesoft.dw.xml_reader.parseDtd`|
+When this property is set to `true`, DataWeave parses a doctype declaration.
+* Type: `Boolean`
+* Default: `false`
+Available for language levels: 2.5
+//mule.dw.remove_shadowed_implicit_inputs//
+|`mule.dw.remove_shadowed_implicit_inputs`|
+When enabled, DataWeave will remove implicit inputs when a variable with the same name is declared at the root level.
```

Comment:
```suggestion
When this property is set to `true`, DataWeave removes implicit inputs if a variable with the same name is declared at the root level.
```

Reply:
Changed wording bc we're enabling the property, not DataWeave itself. I admit it's wordy.

Reply:
added to master